compute_endpoint/tests/integration/endpoint/endpoint/test_interchange.py

```diff
@@ -31,6 +32,45 @@ def reset_signals_auto(reset_signals):
     yield
 
 
+@pytest.fixture
+def mock_log(mocker):
+    yield mocker.patch(f"{_MOCK_BASE}log")
+
+
+@pytest.fixture
+def mock_ex(mocker, endpoint_uuid):
+    ex = mocker.Mock(endpoint_id=endpoint_uuid, executor_exception=None)
+    yield ex
+
+
+@pytest.fixture
+def mock_conf(mocker, mock_ex):
+    conf = Config(executors=[mock_ex])
+    yield conf
+
+
+@pytest.fixture
+def ei_factory(endpoint_uuid, mock_conf):
+    to_stop: t.List[EndpointInterchange] = []
+
+    def _f(*a, **k):
+        reg_info = {"task_queue_info": {}, "result_queue_info": {}}
+        kw = {"endpoint_id": endpoint_uuid, "config": mock_conf, "reg_info": reg_info}
+        kw.update(k)
+        to_stop.append(EndpointInterchange(*a, **kw))
+        return to_stop[-1]
+
+    yield _f
+
+    for _ei in to_stop:
+        _ei.stop()
+
+
+@pytest.fixture
+def ei(ei_factory):
```

Comment:
I recommend renaming to something like `ep_ix` for some additional clarity.

Reply:
Done.